movable.lua

```diff
@@ -463,8 +463,7 @@ do
 		if(not target:GetCenter()) then return end
 		if(backdropPool[target]) then return backdropPool[target] end
 
-		local backdrop = CreateFrame"Frame"
-		backdrop:SetParent(UIParent)
+		local backdrop = CreateFrame("Frame", nil, UIParent, BackdropTemplateMixin and "BackdropTemplate")
```

Comment:
No need to check that the backdrop mixin exists. Same change can be made to the others.
```suggestion
		local backdrop = CreateFrame("Frame", nil, UIParent, "BackdropTemplate")
```

Reply:
I don't check for the existence of the Mixin in my own code, but isn't the goal of doing that making it retro-compatible?